docs/querying/post-aggregations.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: post-aggregations
-title: "Post-Aggregations"
+title: "Postaggregations"
```

Comment:
The title looks strange as one word. Is `postaggregation` a word?
```suggestion
title: "Post Aggregations"
```

Reply:
How about "Post-aggregations".